xml/System.Text.Json/JsonSerializerOptions.xml

```diff
@@ -286,9 +405,14 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Defines whether JSON should pretty print which includes: indenting nested JSON tokens, adding new lines, and adding white space between property names and values. By default, the JSON is serialized without any extra white space.</summary>
         <value>To be added.</value>
```

Comment:
```suggestion
         <value><see langword="true"/> if JSON should pretty print on serialization; otherwise, <see langword="false"/>. The default is <see langword="false"/>.</value>
```